corehq/apps/reports/templates/reports/project_health/partials/last_month_stats.html

```diff
@@ -1,47 +0,0 @@
-{% load hq_shared_tags %}
-{% load i18n %}
-<h2>{% trans "Last Month Stats" %}</h2>
-<table class="table lead">
-  <thead>
-  <tr>
-    <th>{% trans "Category" %}</th>
-    <th>{{ last_month.month|date:"F 'y" }}</th>
-    <th>
-      {% blocktrans with prev_month=last_month.previous_month|date:"F" %}
-        Change from {{ prev_month }}
-      {% endblocktrans %}</th>
-  </tr>
-  </thead>
-  <tr>
-    <td>{% trans "High Performing Users" %}
-      <p class="text-muted"><small>{% blocktrans %}Users who submitted {{ threshold }} or more forms last month.{% endblocktrans %}</small></p>
-    </td>
-    <td>{{ last_month.number_of_performing_users }}</td>
-    <td>
-      {% chevron last_month.delta_high_performing_pct %}
-      {{ last_month.delta_high_performing_pct|floatformat }} %
```

Comment:
@orangejenny more specifically it was failing at this line. @sravfeyn recently tried a fix that made sense in https://github.com/dimagi/commcare-hq/pull/28102, changing it from a totally unreferenced variable to this variable that's defined elsewhere, but the overall issue turned out to be that this code simply hadn't just been silently failing on prod for years, so trying to fix it now seemed pointless

Reply:
I see. I went back to read the ticket and saw [this error](https://sentry.io/organizations/dimagi/issues/1782174450/) linked which is a python 3 change, so it would have been broken since last June, not for years. But it sounds like there's also an issue with an undefined variable. But in any case, it's been broken for a while.

Coincidentally, the approximate python3 rollout date is fixed in my head because it caused another comparison type error that silently failed in a template and broke our analytics for a few weeks. I'd like to see django altogether stop failing silently on template errors.

Reply:
> I'd like to see django altogether stop failing silently on template errors.

+1000 This behavior is not Pythonic!

Reply:
> I'd like to see django altogether stop failing silently on template errors.

Yeah this seems to be Django's plan as well, and exactly what we bumped into here. Going forward if something fails within the `{% include %}`, you get an error (whether `DEBUG` is `True` or `False`)